mailpoet/assets/js/src/subscribers/importExport/export.ts

```diff
@@ -158,7 +158,7 @@ jQuery(document).ready(() => {
       MailPoet.Modal.loading(false);
     }).done((response) => {
       const resultMessage = MailPoet.I18n.t('exportMessage')
-        .replace('%1$s', `<strong>${parseInt(response.data.totalExported, 10).toLocaleString()}</strong>`)
+        .replace('%1$s', `<strong>${parseInt(<string> response.data.totalExported, 10).toLocaleString()}</strong>`)
```

Comment:
Doesn't `toLocaleString` already return string for sure?

Reply:
The problem here is `parseInt` expecting string ([playground](https://www.typescriptlang.org/play?jsx=0&ts=4.5.4#code/EQVwzgpgBGAuBOBLAxrYBuAUAGwrKAZgPZEBcIAdgNYVEDuFUUAvFAIwBMAzFpgA4BDeJACSFWAApiRADTsADAEosg4RDGSAPHCQUA5gD5pctkvRA))

Reply:
Ah, right, I misread the brackets.